Update manual ip for >18.04 (#2)
IBCNServices/jekyll-docs-template: _posts/0000-00-00-manual-public-ip.md

*Use these instructions to assign the public IP to a VMWare virtual machine. For MAAS, see below.*

> Check if the interface `ens224` exists on the VM. If it does not exist, contact Merlijn, Sander or Gregory to move your VM into the DMZ and the interface will be added to the VM.

Reviewer: ```suggestion
Check if the interface `ens224` exists on the VM. If it does not exist, contact Sander (`sborny` on the IDLab Mattermost) to connect your VM to the DMZ and the interface will be added to the VM.
```

Reviewer: Misschien waarschuwen dat we de machine gaan moeten heropstarten om dit te doen?

Reviewer: Zet er gerust uw email als ge liever hebt dat mensen een email sturen.

```bash
ifconfig -a | grep -q ens224 && echo "interface ens224 exists!" || echo "interface ens224 NOT found"
```

> **Ubuntu version >= 18.04**

Reviewer: How can they check their Ubuntu version?

Create the file `/etc/netplan/60-tengu.yaml` and copy the following config, fill in the public ip from jFed.

```yaml
network:
  version: 2
  ethernets:
    ens224:
      dhcp4: no
      addresses: [<PUBLIC_JFED_IP>/26]
      gateway4: 193.190.127.129
```

> Bring up the interface with `sudo netplan try`. If all goes well you should receive the following output. Press <kbd>Enter</kbd> to accept the new configuration.

Reviewer: Maybe also explain that config will be reverted if you don't confirm after 120 seconds, so if they lose connection to the machine, it will revert back?
